Toolkit/ArcGISToolkit/MeasureToolbar.swift

```diff
@@ -407,27 +414,49 @@ public class MeasureToolbar: UIToolbar, AGSGeoViewTouchDelegate {
         mapView?.sketchEditor?.clearGeometry()
     }
     
+    private lazy var linearUnits: [AGSLinearUnit] = {
+        let linearUnitIDs: [AGSLinearUnitID] = [.centimeters, .feet, .inches, .kilometers, .meters, .miles, .millimeters, .nauticalMiles, .yards]
+        return linearUnitIDs
+            .compactMap (AGSLinearUnit.init)
```

Comment:
The extra space is causing a linter warning.

<img width="895" alt="image" src="https://user-images.githubusercontent.com/2257493/71744955-aa0b0680-2e1d-11ea-8ef2-3e658a0c0dd2.png">

```suggestion
            .compactMap(AGSLinearUnit.init)
```